src/Protocol/ActivityPub/Processor.php

```diff
@@ -898,7 +901,7 @@ private static function postMail($activity, $item)
 	 * @return string fetched message URL
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	public static function fetchMissingActivity(string $url, array $child = [], string $relay_actor = '')
+	public static function fetchMissingActivity(string $url, array $child = [], string $relay_actor = '', int $completion = Receiver::COMPLETION_MANUAL)
```

Comment:
Please add the DocBlock entry for this new parameter including a reference to the constants describing its possible values.

Reply:
Suggestion: `One of the self::COMPLETION_* constant values`

Reply:
Done